### PR TITLE
yaml: Merge scc tests into one

### PR DESCRIPTION
**Pull request by dzedro**
Merge https://openqa.suse.de/tests/4357383 and https://openqa.suse.de/tests/4357387

- Verification run:
http://10.100.12.155/tests/16105 12-SP3
http://10.100.12.155/tests/16109 12-SP4
http://10.100.12.155/tests/16108 12-SP5
http://10.100.12.155/tests/16106 15
http://10.100.12.155/tests/16107 15-SP1

### CODE REVIEW

File: schedule/qam/common/qam-scc.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- autoyast/console
+- autoyast/autoyast_reboot
+- installation/grub_test
+- installation/first_boot
+- console/system_prepare
+- console/consoletest_setup
+- qa_automation/patch_and_reboot
+- console/yast2_registration
+- console/suseconnect
+...
```
Comment (rwx788) [inline schedule/qam/common/qam-scc.yaml:18]: Is that on purpose? Looks unintentional.
Also quick questions, will it be parsed correctly without indentation for the list?

Comment (rwx788) [reply schedule/qam/common/qam-scc.yaml:18]: Ah, have found that it's in the standard, learnt something! So ignore first question =)

Comment (dzedro) [reply schedule/qam/common/qam-scc.yaml:18]: not sure if you mean `...`, but yes I like to use it as we yal does start with `---` and making it end with `...` make end obvious
Added indentation, many yamls don't have any indentation

Comment (perlpunk) [reply schedule/qam/common/qam-scc.yaml:18]: JFYI, so called "zero-indented" sequences will be parsed by all known YAML parsers (haven't found one that doesn't (except the old YAML.pm which I fixed ;-))

**Pull request merged**